spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
Ah ok I get it now :D it's a client concern. Though in the documentation "default" doesn't mean that much to me and I didn't jump to that conclusion of "there might be other response types not described here"

So from the point of view of documenting and being kind of nice about it.. would it be ok if we specified the ones we know and care about (200, 401) and also make sure that they all have a default entry like before? I feel like that will 'read' nicer to the user looking at the docs.

Talked a bit about this in slack. But just documenting here.